src/Components/Toggle.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Switch from '@mui/material/Switch';
+import { alpha, styled } from '@mui/material/styles';
+import { grey } from '@mui/material/colors';
+
+const Toggle = styled(Switch)(({ theme }) => ({
```

Comment:
(sorry for comment down here.. gh won't let me make file comment...)

pls move useStyles to end, convert const arrow to function and move styles from inline to useStyles section.